src/System.Runtime.Extensions/tests/System/StringComparer.netcoreapp1.1.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class MiscStringComparerTests
+    {
+        public static object[][] FromComparison_TestData = new object[][] {
```

Comment:
Nit: The field should be `readonly` and `new object[][]` is unnecessary and can be removed.